docs/config.md

```diff
@@ -0,0 +1,138 @@
+# Configuring the Onfido Identity Verification Connector
+
+To use the Onfido Identity Verification Connector with WSO2 Identity Server, you need to configure the connector within 
+the WSO2 Identity Server. Follow the instructions below to set up the Onfido connector using a 
+sample application.
+
+To test this setup, your organization must first have an Onfido account. You will then need to create 
+a workflow in the Onfido Studio and generate the necessary tokens.
+
+## Prerequisites
+
+1. You need to have an Onfido account for this connector to work. 
+Please [contact](https://www.evidentid.com/contact-sales/) the Onfido team and they will be happy to help.
+2. This version of the connector is tested with WSO2 Identity Server version 7.0. 
+Make sure to download and set up the correct version of the
+[Identity Server](https://wso2.com/identity-and-access-management) in your environment.
+
+## Setting up and Installing the Onfido Connector
+
+### Step 1: Extracting the Project Artifacts
+
+1. Clone the `identity-verification-onfido` repository.
+2. Build the project by running `mvn clean install` in the root directory.
+
+**Note:** The latest project artifacts can also be downloaded from the Connector Store.
+
+### Step 2: Deploying the Onfido Authenticator
+
+1. Navigate to `identity-verification-onfido/components → org.wso2.carbon.identity.verification.onfido.connector → target`.
+2. Copy the `org.wso2.carbon.identity.verification.onfido.connector-<version>-SNAPSHOT.jar` file.
+3. Navigate to `<IS_HOME>/repository/components/dropins`.
+4. Paste the `.jar` file into the dropins directory.
+5. Alternatively, you can drag and drop the `.jar` file into the dropins directory.
+6. Next, navigate to `identity-verification-onfido/ui-metadata` and copy the `onfido` directory.
+7. Paste it into `<IS_HOME>/repository/resources/identity/extensions/identity-verification-providers` directory.
+
+### Step 3: Deploying the Onfido Webhook REST API
+
+1. Navigate to `identity-verification-onfido/components → org.wso2.carbon.identity.verification.onfido.api → org.wso2.carbon.identity.verification.onfido.api.dispatcher → target`.
+2. Copy the `idv#onfido.war` file.
+3. Navigate to `<IS_HOME>/repository/deployment/server/webapps`.
+4. Paste the `.war` file into the webapps directory.
+5. Next, navigate to `<IS_HOME>/repository/conf`.
+6. Open the `deployment.toml` file.
+7. Add the following lines of code:
+
+    ```toml
+    [[resource.access_control]]
+    context = "(.*)/idv/onfido/v1/(.*)/verify"
+    secure = "false"
+    http_method = "POST"
+
+    [tenant_context]
+    enable_tenant_qualified_urls = "true"
+    enable_tenanted_sessions = "true"
+    rewrite.custom_webapps = ["/idv/onfido/"]
+    ```
+
+## Configuring Onfido Identity Verification Provider in WSO2 Identity Server Console
```

Comment:
The contents related to configuring onfido in the console needs to be changed according to the updated UI.

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/11/commits/e366865a038d05794bd3ffdaf1dbdce621a239c0